app/styles/ui/_side-by-side-diff.scss

```diff
@@ -0,0 +1,284 @@
+      direction: rtl;
+    }
+
+    &.selectable {
+      span {
+        cursor: pointer;
+      }
+
+      &.line-selected {
+        background: var(--diff-selected-background-color);
+        color: var(--diff-selected-text-color);
+        border-color: var(--diff-selected-border-color);
+
+        &:hover,
+        &.hover {
+          background: var(--diff-hover-background-color);
+          color: var(--diff-hover-text-color);
+          border-color: var(--diff-hover-border-color);
+        }
+      }
+    }
+  }
+
+  .hunk-handle {
+    position: absolute;
+    height: 100%;
+    width: 20px;
+    left: calc(50% - 10px);
+    top: 0;
+    cursor: pointer;
+  }
+
+  .content {
+    padding-left: var(--spacing);
+    white-space: pre-wrap;
+    overflow-y: auto;
+    flex-grow: 1;
+    word-break: break-all;
+
+    &,
+    * {
+      cursor: text;
+      user-select: text;
+    }
+
+    &:before {
+      content: ' ';
+      padding-right: var(--spacing);
+    }
+
+    .octicon {
+      height: 8px;
+      fill: var(--error-color);
+      display: inline-block;
+      margin-left: 3px;
+      margin-bottom: -1px;
+    }
+  }
+
+  .before,
+  .after {
+    width: 50%;
+    display: flex;
+    flex-direction: row;
+  }
+
+  .before {
+    color: var(--diff-delete-text-color);
+    background: var(--diff-delete-background-color);
+    border-right: 1px solid var(--diff-border-color);
+
+    .line-number {
+      background: var(--diff-delete-gutter-background-color);
+      border-color: var(--diff-delete-border-color);
+    }
+
+    .selectable:hover,
+    .selectable.hover {
+      background: var(--diff-delete-hover-background-color);
+      color: var(--diff-delete-hover-text-color);
+      border-color: var(--diff-delete-hover-border-color);
+    }
+  }
+
+  .after {
+    color: var(--diff-add-text-color);
+    background: var(--diff-add-background-color);
+
+    .line-number {
+      background: var(--diff-add-gutter-background-color);
+      border-color: var(--diff-add-border-color);
+    }
+
+    .selectable:hover,
+    .selectable.hover {
+      background: var(--diff-add-hover-background-color);
+      color: var(--diff-add-hover-text-color);
+      border-color: var(--diff-add-hover-border-color);
+    }
+  }
+
+  .row {
+    display: flex;
+    flex-direction: row;
+    line-height: 20px;
+    position: relative;
+
+    &.hunk-info {
+      background: var(--diff-hunk-background-color);
+      color: var(--diff-hunk-text-color);
+
+      div,
+      span {
+        user-select: none;
+        cursor: default;
+      }
+
+      .line-number {
+        background: var(--diff-hunk-background-color);
+        border-color: var(--diff-border-color);
+      }
+    }
+
+    &.deleted .before {
+      width: calc(50% + 4px);
+
+      .line-number {
+        border-right-width: 4px;
+      }
+    }
+
+    &.added .after {
+      width: calc(50% + 4px);
+
+      .line-number {
+        border-left-width: 4px;
+      }
+    }
```

Comment:
These changes seem to have caused a 1px misalignment

![image](https://user-images.githubusercontent.com/634063/94442593-7094db00-01a4-11eb-924e-13cdc6eac700.png)